### DISCUSSION TITLE
每次启动fe都有大量add SCHEMA_CHANGE job的日志

### DISCUSSION THREAD

**Discussion by lgq19991225** [apache/doris#20859] [A - General / Q&A]
每次启动fe都有大量add SCHEMA_CHANGE job的日志，fe等到job完成才可用，重新启动还是会有这些日志，doris版本1.2.4.1
![image](https://github.com/apache/doris/assets/34154643/1c293e66-fb04-410f-b315-91510fbc1c1b)

**Comment by yagagagaga** [comment]
你之前有做过大量的 schema change 吗？

**Reply by lgq19991225** [reply]
有做过，我发现metadata_failure_recovery一直是开着的，会不会是这个原因

**Reply by LemonLiTree** [reply]
metadata_failure_recovery 关闭
